client/src/pages/ViewRequestStatus/RequestStatusView.js

```diff
@@ -87,7 +95,9 @@ export function RequestStatus() {
 			method: "DELETE",
 		}).then((res) => {
 			if (res.status === 201) {
-				new Notification("You have been assigned another available laptop");
+				if ("Notification" in window && Notification.permission === "granted") {
+					new Notification("You have been assigned another available laptop");
```

Comment:
Just from reading the code, I'm not 100% sure this works: I think it would at least need to set reloadNeeded. And I would not rely on the unusual "DELETE returns 201" to know that a new assignment had been made, instead, I would use the socket. I really like the differentiated message though ("another available laptop")